net-print/fax4cups/fax4cups-1.29-r1.ebuild

```diff
@@ -23,27 +23,26 @@ RDEPEND="${DEPEND}
 	)
 	app-admin/sudo
 "
-
-S=${WORKDIR}/fax4CUPS-${PV}
+BDEPEND="net-print/cups"
 
 src_install() {
 	doman fax4CUPS.1
 
-	exeinto $(cups-config --serverbin)/backend
+	exeinto $(cups-config --serverbin)/backend || die
```

Comment:
shouldn't the `|| die` go into the `$()`?

Reply:
yes, my bad